### ISSUE TITLE
Usability: Icons for Follow and Unfollow are so similar I thought they were the same at first

### ISSUE THREAD

**Issue by pathway** [mastodon/mastodon#20028]
### Pitch

Blue person outline with "plus" (follow) ,  is far too similar to blue person outline with "x" (unfollow) 
So we make the unfollow one red. (or just the x red, or really any other visually obvious difference)

### Motivation

People who arent looking super close or have imperfect vision, can get the opposite result (unfollowing) to what they want (following)

**Comment by placoderm** [comment]
I agree with this. But I don't have a good solution. Perhaps if the + and x were about twice as large?

**Comment by andreacpowers** [comment]
1. Use words instead of symbols.
2. The icon now shows status, not action. Change it to show action. I get confused all the time with the icon now. Does the + mean I'm following them or does it mean click me to switch to unfollow?
